bin/node/runtime/pangolin/src/bridges/substrate/millau_messages.rs

```diff
@@ -247,3 +248,16 @@ impl SourceHeaderChain<bp_millau::Balance> for Millau {
 		)
 	}
 }
+
+pub struct MillauCallToPayload;
+
+impl CallToPayload<ToMillauMessagePayload> for MillauCallToPayload {
+	fn to_payload(spec_version: u32, call: Vec<u8>) -> ToMillauMessagePayload {
+		return FromThisChainMessagePayload::<WithMillauMessageBridge> {
+			spec_version,
+			weight: 100,
```

Comment:
Is 100 enough for this `weight`?

 If it is less than actual static weight, the call is not dispatched. What's the actual static weight then?
```
/// Message payload type used by dispatch module.
#[derive(RuntimeDebug, Encode, Decode, Clone, PartialEq, Eq)]
pub struct MessagePayload<SourceChainAccountId, TargetChainAccountPublic, TargetChainSignature, Call> {
	/// Runtime specification version. We only dispatch messages that have the same
	/// runtime version. Otherwise we risk to misinterpret encoded calls.
	pub spec_version: SpecVersion,
	/// Weight of the call, declared by the message sender. If it is less than actual
	/// static weight, the call is not dispatched.
	pub weight: Weight,
	/// Call origin to be used during dispatch.
	pub origin: CallOrigin<SourceChainAccountId, TargetChainAccountPublic, TargetChainSignature>,
	/// The call itself.
	pub call: Call,
}
```